code/modules/research/techweb/all_nodes.dm

```diff
@@ -34,6 +34,7 @@
 		"coffee_cartridge",
 		"coffeemaker",
 		"coffeepot",
+		"syrup_bottle",
```

Comment:
ABC order

Reply:
done in [1758d26](https://github.com/tgstation/tgstation/pull/70991/commits/1758d260bf38349a463fb0c4c94b19bde1396d9f)